core/src/main/java/com/scalar/db/common/StorageInfoImpl.java

```diff
@@ -11,14 +11,17 @@ public class StorageInfoImpl implements StorageInfo {
   private final String storageName;
   private final MutationAtomicityUnit mutationAtomicityUnit;
   private final int maxAtomicMutationsCount;
+  private final boolean consistentVirtualTableRead;
```

Comment:
```suggestion
  private final boolean canConsistentVirtualTableRead;
```

Reply:
Fixed in Fixed in 92e5729d112ae559b0a8d7230e74aa78f7cf00e3. Thanks!